connect-and-run.md

```diff
@@ -144,6 +144,19 @@ The parameters listed below make up the profile settings for configuring a conne
 
 Open the command palette (`F1`, or `Ctrl+Shift+P` on Windows or Linux, or `Shift+CMD+P` on OSX). After executing the `SAS.addProfile` command, select the SAS 9.4 (remote) connection type and complete the prompts (using values from the preceeding table) to create a new profile.
 
+## Profile Anatomy (SAS 9.4 local)
+
+The parameters listed below make up the profile settings for configuring a connection to a local SAS 9.4 instance.
+
+| Name     | Description                      | Additional Notes                    |
+| -------- | -------------------------------- | ----------------------------------- |
+| **Name** | Name of the profile              | This will display on the status bar |
+| **Host** | Indicates SAS 9.4 local instance | Defaults to localhost for com       |
+
+## Add New SAS 9.4 Local Profile
+
+Open the command palette (`F1`, or `Ctrl+Shift+P` on Windows or Linux, or `Shift+CMD+P` on OSX). After executing the `SAS.addProfile` command, select the SAS 9.4 (local - COM) connection type to create a new profile.
```

Comment:
COM connection profile is only supported on Windows. Can we remove the shortcuts for linux and osx?